src/admin/tests.py

```diff
@@ -88,7 +88,7 @@ def test_length(self):
         self.client.force_authenticate(user=self.user)
 
         response = self.client.get(reverse("self-check"))
-        self.assertEqual(len(response.data["d"]), 14)
+        self.assertEqual(len(response.data["d"]), 17)
```

Comment:
I feel like this is a needlessly brittle test but this can be changed later

Reply:
Honestly not sure what this even does but didn't care enough to figure it out.

Reply:
Thanks Daniel, glad to hear you have confidence in your code.